.circleci/config.yml

```
@@ -4,12 +4,14 @@ workflows:
   version: 2
   build_and_test_all:
     jobs:
-      - build-test-linux
+      - build-test-linux:
+          context:
+            - hello-world-demos
 
 jobs:
   build-test-linux:
     docker:
-      - image: circleci/python:2 # same image restwrappers use
+      - image: cimg/python:3.9.9
```

Since we don't use Python as part of the build, I think this can be [cimg/base](https://circleci.com/developer/images/image/cimg/base):
```suggestion
      - image: cimg/base:2022.04
```